tests/test-office-formats.sh

```
@@ -219,7 +222,7 @@ unzip -q "$zip_file" "$docx_filename" -d "$temp_dir"
 
 # Verify the extracted DOCX is a valid ZIP archive
 unzip -t "$temp_dir/$docx_filename" > /dev/null 2>&1
-if [ $? -eq 0 ]; then
+if [[ $? -eq 0 ]]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)